config.js

```diff
@@ -1016,6 +1016,8 @@ config.NSFS_LIST_IGNORE_ENTRY_ON_EACCES = true;
 // we will for now handle the same way also EINVAL error - for gpfs stat issues on list (.snapshots)
 config.NSFS_LIST_IGNORE_ENTRY_ON_EINVAL = true;
 
+config.NSFS_USER_BUCKET_PATH_HTTP_HEADER = 'x-nsfs-bucket-path';
```

Comment:
@jackyalbo I would use x-noobaa- as a prefix for anything that is noobaa specific
```suggestion
config.NSFS_USER_BUCKET_PATH_HTTP_HEADER = 'x-noobaa-nsfs-bucket-path';
```